src/components/widgets-settings/attach-kpi/attach-kpi.component.coffee

```diff
@@ -0,0 +1,95 @@
+module = angular.module('impac.components.widgets-settings.attach-kpis', [])
+module.component('settingAttachKpi', {
+  bindings: {
+    widget: '<'
+    chartPromise: '<?'
+    options: '<?'
+    onInit: '&?'
+    onComplete: '&?'
+  }
+  templateUrl: 'widgets-settings/attach-kpi.tmpl.html'
+  controller: ($timeout, ImpacKpisSvc, ImpacDashboardsSvc)->
+    ctrl = this
+
+    ctrl.$onInit = ->
+      # Define properties
+      # -------
+      ctrl.kpi = {}
+      ctrl.showPanel = false
+      ctrl.draftTarget = ''
+      ctrl.options ||= {
+        chartShrinkSize: 30
+      }
+      ctrl.dhbCurrency = ImpacDashboardsSvc.getCurrentDashboard().currency
```

Comment:
should be passed to the directive/component as an argument => the widget has a currency on its metadata that could be used